swc/private/swc.bzl

```diff
@@ -163,10 +163,14 @@ def _impl(ctx):
                 ),
             )
 
+    # See https://docs.bazel.build/versions/main/skylark/rules.html#runfiles
+    runfiles = ctx.runfiles(files = outputs + ctx.files.data)
+    runfiles = runfiles.merge_all([d[DefaultInfo].default_runfiles for d in ctx.attr.data])
```

Comment:
Will it always have a `DefaultInfo` provider?

Reply:
hmm, yes I think every rule does?